cli/edit.go

```diff
@@ -41,3 +45,53 @@ func editProjectCommand(t *core.Timetrace) *cobra.Command {
 
 	return editProject
 }
+
+type editOptions struct {
+	Plus  string
+	Minus string
+}
+
+func editRecordCommand(t *core.Timetrace) *cobra.Command {
+	var options editOptions
+
+	editRecord := &cobra.Command{
+		Use:   "record <KEY>",
+		Short: "Edit a record",
+		Args:  cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			layout := defaultRecordArgLayout
+
+			if t.Config().Use12Hours {
+				layout = "2006-01-02-03-04PM"
+			}
+
+			recordTime, err := time.Parse(layout, args[0])
+			if err != nil {
+				out.Err("Failed to parse date argument: %s", err.Error())
+				return
+			}
+
+			if options.Minus == "" && options.Plus == "" {
+				out.Info("Opening %s in default editor", recordTime)
+				if err := t.EditRecordManual(recordTime); err != nil {
+					out.Err("Failed to edit project: %s", err.Error())
+					return
+				}
+			} else if options.Plus != "" && options.Minus != "" {
+				out.Err("Plus and minus flag can not be combined: %s", errors.New("edit not possible"))
```

Comment:
```suggestion
			} else if options.Plus != "" && options.Minus != "" {
				out.Err("Plus and minus flag can not be combined: %s", errors.New("edit not possible"))
				return
```

I think we should return in that case.

Reply:
It would also be possible to check `if options.Plus != "" && options.Minus != ""` right at the beginning - that probably would simplify this if-else-block here.

Reply:
Hey, thanks for the feedback! You are completely right, I just forgot the return statement. I put the flag-option check now at the beginning of the method to simplify the method in my next commit.